MAUI/DataGrid/selection.md

```diff
@@ -383,6 +386,130 @@ The selected rows will be deleted.
 </tr>
 </table>
 
+## Multiple Row or Cell Selection
+
+The `SfDataGrid` allows you to select multiple rows or cells by setting `SelectionMode` property as [Extended](), where you can select multiple rows or cells by dragging the mouse on SfDataGrid and also using the key modifiers.
+
+While using `Extended`, you can select multiple rows or cells by pressing the key modifiers <kbd>Ctrl</kbd> and <kbd>Shift</kbd>. 
```

Comment:
The SfDataGrid allows you to select multiple rows or cells by setting the SelectionMode property to Extended. This enables selection of multiple rows or cells by dragging the mouse or by clicking while holding modifier keys such as Ctrl and Shift.